src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -187,6 +191,16 @@ function MoneyRequestPreview(props) {
     };
 
     const getPreviewHeaderText = () => {
+        if (isExpensifyCardTransaction) {
```

Comment:
@fedirjh I moved the logic down below distance to fix the distance requests, I think it's better to keep each type of request separate since that's the pattern so far.